forward:
  power: 0
  duration: 0
up:
  power: 0
  duration: 0
right:
  power: 0
  duration: 0
light_mode: 0
speed_mode: 0